pkg/service/interfaces.go

```diff
@@ -13,6 +13,7 @@ import (
 //counterfeiter:generate . RoomStore
 type RoomStore interface {
```

Comment:
should this change too? maybe ObjectStore?

Reply:
yeah, I considered it but couldn't think of a good new name for RORoomStore - it's used by RoomService and RTCService, maybe `ServiceStore`?

Reply:
`ROStore` ?